client/src/components/directory/subComponents/DirectoryTable.jsx

```diff
@@ -92,6 +96,33 @@ class DirectoryTable extends Component {
                         })
                       }
                     </TableCell>
+                    <TableCell>
```

Comment:
We should move this condition out (`scheduledCandidateIds && scheduledCandidateIds.includes(row.id)` ? <TableCell /> : null`), otherwise other tables that don't have this will render an empty column